csharp/ql/src/Bad Practices/PathCombine.ql

```diff
@@ -1,6 +1,7 @@
 /**
- * @name Call to System.IO.Path.Combine
- * @description Finds calls to System.IO.Path's Combine method
+ * @name Calls to System.IO.Path.Combine may silently drop its earlier arguments
+ * @description Path.Combine may silently drop its earlier arguments
+ *              if its later arguments are absolute paths.
```

Comment:
```suggestion
 * @name Call to 'System.IO.Path.Combine' may silently drop its earlier arguments
 * @description 'Path.Combine' may silently drop its earlier arguments
 *              if its later arguments are absolute paths.
```

(I realize this was also the case before, but we may as well adhere to the [style guide](https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md) when we are updating it.)